lib/main.dart

```diff
@@ -42,6 +44,7 @@ class MyApp extends StatelessWidget {
   }
 }
 
+// Carlo
```

Comment:
Good practice for code organisation to implement SplashScreen as a separate widget.